src/utils/CompilerLocationUtils.fs

```diff
@@ -189,6 +189,10 @@ module internal FSharpEnvironment =
         // Check for an app.config setting to redirect the default compiler location
         // Like fsharp-compiler-location
         try
+            // We let you set FSHARP_COMPILER_BIN. I've rarely seen this used and its not documented in the install instructions.
+            match Environment.GetEnvironmentVariable("FSHARP_COMPILER_BIN") with
+            | result when not (String.IsNullOrWhiteSpace result) -> Some result
+            |_->
```

Comment:
I would prefer if you intend the following code block to prevent any confusion.

Reply:
I was following the style already in use there which followed the pattern

    match ... with
    | Some .. -> ....
    | _ ->
    ...
    match ... with
    | Some .. -> ....
    | _ ->

Although admittedly in the end it broke it by indenting the final `if then else`

If you still prefer multiple indents I will change that section so its consistent.